pkg/apiserver/config.go

```diff
@@ -16,6 +16,44 @@ limitations under the License.
 
 package apiserver
 
+import (
+	"k8s.io/apimachinery/pkg/apimachinery/announced"
+	"k8s.io/apimachinery/pkg/apimachinery/registered"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/install"
+)
+
+var (
+	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
+	registry             = registered.NewOrDie("")
+	// Scheme for API object types
+	Scheme = runtime.NewScheme()
+	// Codecs for creating a server config
+	Codecs = serializer.NewCodecFactory(Scheme)
+)
+
+func init() {
```

Comment:
It's a constant/global for the binary.  Not having an init block just means that you can have a uninitialized value that creates unusual behavior until you wrap a method in a sync.Once and then have every caller to your package call it.  That's nuts.

Alternatively, you could plumb it as an arg everywhere that needs it, which is a valid solution, but a level of purity we hadn't reached in the apiserver refactors yet.